docs/about/cedar-design-system/README.md

```diff
@@ -14,45 +14,71 @@
 
-## About Cedar
-Cedar is REI’s open-source Design System for digital products and experiences. Cedar includes the resources to create user interfaces consistent with REI’s [experience principles](../../foundation/experience-principles/), brand styles, and best practices.
+ A design system is a collection of reusable UI components and guidance to help digital teams make design and development decisions. 
+  
+## Meet Cedar
+Cedar is REI’s open-source design system for digital products and experiences, based on REI’s visual language. Its purpose is simple: to help people create consistent, unified user interfaces. We do this by simplifying common design decisions, freeing folks to focus on groundbreaking work. 
+  
+Cedar contains the reusable resources for creating consistent user interfaces informed by REI's [experience principles](../../foundation/experience-principles/), brand styles, and best practices.
 
-As a foundation for digital product development, the Cedar Design System provides a set of reusable components, style variables, design assets, and guidance for teams to efficiently build high-quality and accessible user interfaces.
+Cedar is a living system, meant to grow and evolve to meet the needs of the REI digital community.
 
-Cedar is maintained by a core team of designers, developers, and writers that are dedicated to developing and supporting the system. Cedar welcomes collaboration with the REI digital community to help evolve the system to suit changing product needs.
 
+### Open source
 
-#### Open Source
+Cedar is aligned with our values as a co-op built on community. While Cedar is primarily focused on serving teams building experiences for REI, the system is also available as an open-source project in GitHub and NPM.
 
-While Cedar is primarily focused on serving teams building experiences for REI, the system is also available as an open source project in GitHub and NPM.
+Sometimes there are exceptions to our open-source rule. For example, fonts used in Cedar are licensed and proprietary to REI, and are not for common use unless approved.
 
-Fonts used in the Cedar Design System are licensed and proprietary to REI, and are not for usage by unapproved parties.
+We encourage you to explore Cedar! For more information or questions regarding Cedar’s support for external consumers using the open-source project, please reach out to [cedar@rei.com](mailto:cedar@rei.com).  
 
-For more information or questions regarding Cedar’s support for external consumers using the open source project, please reach out to [cedar@rei.com](mailto:cedar@rei.com).  
 
+## Cedar's parts
+  
+Cedar provides a set of reusable components, style variables, design assets, and guidance for teams to efficiently build high-quality and accessible user interfaces.
 
-## Intake
+### Designer toolkit
+The Cedar UI toolkit is distributed through shared libraries in Figma, and includes all published components and foundational styles.
```

Comment:
this will need a space between the heading and the content for the header to render